python/ray/tune/trial_runner.py

```diff
@@ -219,7 +220,22 @@ def __init__(self,
         if max_pending_trials == "auto":
             # Auto detect
             if isinstance(self._search_alg, BasicVariantGenerator):
-                self._max_pending_trials = 1000
+                # Use a minimum of 16 to trigger fast autoscaling
+                # Scale up to at most the number of available cluster CPUs
+                cluster_cpus = ray.cluster_resources().get("CPU", 1.)
```

Comment:
maybe you can do something larger than # of cluster cpus, like 1.1x? just to put more pressure on the autoscaler

Reply:
Good point - this will increase overhead very little but might make the difference between triggering one or two nodes to start. Updated.